_includes/tutorials/finding-distinct/kstreams/code/src/main/java/io/confluent/developer/FindDistinctEvents.java

```diff
@@ -30,224 +30,227 @@
-    props.put(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, envProps.getProperty("schema.registry.url"));
-    props.put(StreamsConfig.CACHE_MAX_BYTES_BUFFERING_CONFIG, 0);
-
-    return props;
-  }
-  public static Properties loadEnvProperties(String fileName) throws IOException {
-    Properties envProps = new Properties();
-    FileInputStream input = new FileInputStream(fileName);
-    envProps.load(input);
-    input.close();
-
-    return envProps;
-  }
-
-  public static void main(String[] args) throws IOException {
-    if (args.length < 1) {
-      throw new IllegalArgumentException(
-          "This program takes one argument: the path to an environment configuration file.");
+    public static Properties loadEnvProperties(String fileName) throws IOException {
+        Properties envProps = new Properties();
+        FileInputStream input = new FileInputStream(fileName);
+        envProps.load(input);
+        input.close();
+
+        return envProps;
     }
 
-    new FindDistinctEvents().runRecipe(args[0]);
-  }
-
-  private void runRecipe(final String configPath) throws IOException {
-    Properties envProps = this.loadEnvProperties(configPath);
-    Properties streamProps = this.buildStreamsProperties(envProps);
-
-    Topology topology = this.buildTopology(envProps, this.buildClicksSerde(envProps));
-    this.createTopics(envProps);
-
-    final KafkaStreams streams = new KafkaStreams(topology, streamProps);
-    final CountDownLatch latch = new CountDownLatch(1);
-
-    // Attach shutdown handler to catch Control-C.
-    Runtime.getRuntime().addShutdownHook(new Thread("streams-shutdown-hook") {
-      @Override
-      public void run() {
-        streams.close();
-        latch.countDown();
-      }
-    });
-
-    try {
-      streams.start();
-      latch.await();
-    } catch (Throwable e) {
-      System.exit(1);
+    public static void main(String[] args) throws IOException {
+        if (args.length < 1) {
+            throw new IllegalArgumentException(
+                    "This program takes one argument: the path to an environment configuration file.");
+        }
+
+        new FindDistinctEvents().runRecipe(args[0]);
     }
-    System.exit(0);
 
-  }
+    private void runRecipe(final String configPath) throws IOException {
+        Properties envProps = this.loadEnvProperties(configPath);
+        Properties streamProps = this.buildStreamsProperties(envProps);
+
+        Topology topology = this.buildTopology(envProps, this.buildClicksSerde(envProps));
+        this.createTopics(envProps);
+
+        final KafkaStreams streams = new KafkaStreams(topology, streamProps);
+        final CountDownLatch latch = new CountDownLatch(1);
+
+        // Attach shutdown handler to catch Control-C.
+        Runtime.getRuntime().addShutdownHook(new Thread("streams-shutdown-hook") {
+            @Override
+            public void run() {
+                streams.close();
```

Comment:
Suggestion - I know this is pre-existing code, but can we update `close()` to `close(Duration.ofSeconds(5))`